examples/cookbook/api-keys/near-connection.js

```diff
@@ -0,0 +1,29 @@
+// demonstrates how to use API-KEY with 'connect' function. 
+const { connect, keyStores } = require("near-api-js");
+const path = require("path");
+
+const homedir = require("os").homedir();
+const CREDENTIALS_DIR = ".near-credentials";
+const credentialsPath = path.join(homedir, CREDENTIALS_DIR);
+const keyStore = new keyStores.UnencryptedFileSystemKeyStore(credentialsPath);
+
+const RPC_API_ENDPOINT = 'http://34.141.205.230:3030/';
```

Comment:
does this need to be swapped out with a URL or was this value provided by SRE for the time being?

Reply:
It was provided by our SRE team. Replaced with a placeholder, since we will not post a real working key here anyway.